test/plugins/webhooks.test.js

```diff
@@ -493,6 +494,9 @@ describe('webhooks plugin test', () => {
 
                 return server.inject(options).then((reply) => {
                     assert.equal(reply.statusCode, 201);
+                    assert.calledOnce(pipelineFactoryMock.scm.getCommitRefSha);
+                    assert.calledWith(pipelineFactoryMock.scm.getCommitRefSha,
+                        sinon.match({ refType: 'tags' }));
```

Comment:
Should adding `sinon.match()` to other test cases be better?
It looks correct usage.
https://sinonjs.org/releases/latest/matchers/